test/blueprints/Do.Test.Blueprints.Service.Application/RestApi/Analyzer/Demo.generated.cs

```diff
@@ -0,0 +1,19 @@
+// This file will be auto-generated
+
+using Microsoft.AspNetCore.Mvc;
+
+namespace Do.Test;
+
+[ApiController]
+public class DemoController
```

Comment:
generated files should have a corresponding business object. use `Singleton` instead